### PR TITLE
Updating README.md with subreddit info and link to Code of Conduct

### PR DESCRIPTION
**Pull request by GabLotus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,10 @@ Bevy is still in the _very_ early stages of development. APIs can and will chang
 * **[Bevy Rust API Docs](https://docs.rs/bevy):** Bevy's Rust API docs, which are automatically generated from the doc comments in this repo.
 
 ## Community
+* Before contributing or participating in discussions with the community, <br>
+it's a good thing to get familiar with our **[Code of Conduct](https://github.com/bevyengine/bevy/blob/master/CODE_OF_CONDUCT.md)**
 * **[Discord Server](https://discord.gg/gMUk5Ph):** Bevy's official discord server.
+* **[Subreddit](https://reddit.com/r/bevy):** Bevy's official subreddit.
```
Comment (cart) [inline README.md:38]: Can we change these to:
```
[Discord]
[Reddit]
```

i think thats a bit "friendlier"

Comment (GabLotus) [reply README.md:38]: Sure will fix

**Comment by cart** [comment]
Lets remove the explicit line break and be a bit more explicit:
```
you should familiarize yourself with our Code of Conduct
```

**Comment by GabLotus** [comment]
image
Removing the linebreak has this unfortunate side effect. Up to you I can put it back or leave it as is.

**Comment by GabLotus** [comment]
nevermind

**Pull request merged**